include/vcpkg/base/files.h

```diff
@@ -140,13 +140,14 @@ namespace vcpkg
         ExpectedL<Unit> try_read_all_from(long long offset, void* buffer, std::uint32_t size);
     };
 
+    enum class Append
```

Comment:
This should be in the fwd

Reply:
Done. I also removed some unnecessary included (the fwd header was only included transitively by other not needed includes)